src/gui/programs/waddy.rs

```diff
@@ -542,13 +542,20 @@ impl WaddyGui {
                     let count = self.instances[instance_index].texture_tiles.len();
                     let is_search_enabled = self.instances[instance_index].search.enable;
 
+                    let search_text = if is_search_enabled {
+                        self.instances[instance_index].search.text.to_lowercase()
+                    } else {
+                        String::new()
+                    };
```

Comment:
This might be redundant. Just convert to lowercase without any conditions. Search text always have a valid string and empty string is a valid string.